paper-typeahead-input.html

```diff
@@ -313,7 +340,7 @@
         }
       });
       var input = Polymer.dom(this.root).querySelector('paper-input');
-      input.$.input.autocomplete = 'address-level4';
+      input.$.input.autocomplete = this.autocomplete;
```

Comment:
Got it.   But if I didn't remember wrong, I have been searching for the chrome suggesting problem a lot before and the conclusion of discussions I found is that autocomplete="Off" is not reliable, workaround is to use autocomplete="address-level4".  But it was quite ago, I am not sure if Chrome made some updates to change the situation here.  If you like, can you search for some docs or evidence to say autocomplete="Off" is working now?    Also, you change it to be "Off" is it because the original implementation using  autocomplete="address-level4" is not working?